examples/BUILD

```diff
@@ -111,3 +112,20 @@ cc_fuzz_test(
         "@re2",
     ],
 )
+
+java_fuzz_test(
+    name = "JavaFuzzTest",
```

Comment:
We use lower_case naming conventions for Bazel targets. Should be `java_fuzz_test`.

Also by convention we alphabetize the target names in the BUILD file.

Reply:
> We use lower_case naming conventions for Bazel targets. Should be `java_fuzz_test`.

Instances of `java_binary` and `java_test` are customarily named after the main class, which allows for omitting the `main_class` attribute (`java_library` targets use lower_ case, which lines up with package names in Java). I felt that it would be useful for `java_fuzz_test` to follow the same convention with respect to it's `target_class`. See https://docs.bazel.build/versions/master/be/java.html#java_binary_args for more details.

> Also by convention we alphabetize the target names in the BUILD file.

Will do.

Reply:
> > We use lower_case naming conventions for Bazel targets. Should be `java_fuzz_test`.
> 
> Instances of `java_binary` and `java_test` are customarily named after the main class, which allows for omitting the `main_class` attribute (`java_library` targets use lower_ case, which lines up with package names in Java). I felt that it would be useful for `java_fuzz_test` to follow the same convention with respect to it's `target_class`. See https://docs.bazel.build/versions/master/be/java.html#java_binary_args for more details.

Ah! I understand now, thank you for the extra context.